### PR TITLE
[12.0] [MIG] project_timeline_task_dependency

### PR DESCRIPTION
**Pull request by Nikul-Chaudhary**
Project Timeline Task Dependencies
==================================  
This module shows arrows between tasks and their dependencies (on the timeline view).

### CODE REVIEW

**Comment by Nikul-Chaudhary** [comment]
@pedrobaeza  I added dependency https://github.com/OCA/project/pull/469/files#diff-58bf4ef7bb17e0cabe9c102156155697R2
but Error : https://travis-ci.org/OCA/project/jobs/483037968#L1100

**Comment by pedrobaeza** [comment]
In this case it's not working because it belongs to the same repository.

**Comment by Nikul-Chaudhary** [comment]
@pedrobaeza Ok, We will waiting for merge others

**Comment by pedrobaeza** [comment]
You can rebase now

**Comment by pedrobaeza** [comment]
Well, it's not going to work as `project_timeline` is not merged yet... Sorry for telling so soon.

**Comment by Nikul-Chaudhary** [comment]
> You can rebase now

@pedrobaeza Done

**Comment by pedrobaeza** [comment]
Please rebase

**Comment by Nikul-Chaudhary** [comment]
> Please rebase

rebased

File: oca_dependencies.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 web
+project https://github.com/Nikul-Chaudhary/project 12-mig-project_timeline
```
Comment (pedrobaeza) [inline oca_dependencies.txt]: Please remove this one

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Squash together "[UPD] README.rst" commits

**Review by LoisRForgeFlow — APPROVED** [review]
:+1:

**Pull request merged**